client/payment-request/style.scss

```diff
@@ -11,26 +11,40 @@
 	background-size: contain;
 	border: 0;
 	border-radius: 4px;
-	box-shadow: rgba( 60, 64, 67, 0.3 ) 0 1px 1px 0,
-		rgba( 60, 64, 67, 0.15 ) 0 1px 3px 1px;
 	cursor: pointer;
 	height: 40px;
 	min-height: 40px;
 	padding: 11px 24px;
 	width: 100%;
 
 	&.light {
-		// account for button box-shadow when setting width
+		background-color: #fff;
+		&:active {
+			background-color: #fff;
+		}
+		&:hover {
+			background-color: #f8f8f8;
+		}
+		&:focus {
+			outline: 0;
+		}
+	}
+
+	&.light-outline {
+		// Account for button box-shadow when setting width.
 		width: calc( 100% - 3px );
+		// Add vertical margin to make box-shadow visible in blocks.
+		margin: 2px 0 3px 0;
 		background-color: #fff;
+		box-shadow: rgba( 60, 64, 67, 0.3 ) 0 1px 1px 0,
+			rgba( 60, 64, 67, 0.15 ) 0 1px 3px 1px;
 		&:active {
 			background-color: #fff;
 		}
 		&:hover {
 			background-color: #f8f8f8;
 		}
 		&:focus {
-			box-shadow: #e8e8e8 0 1px 1px 0, #e8e8e8 0 1px 3px;
 			outline: 0;
 		}
```

Comment:
We need to include the `box-shadow` here to indicate whether the element is focused when (for example) navigating via the <kbd>tab</kbd> key.

Personally, I think we should allow the purple outline set by WC Core, since that's really universal throughout WC pages; I don't see why we would remove that. Maybe @LevinMedia has some thoughts on that, design-wise?
The `box-shadow` on its own is **very difficult** to notice.

We should also make sure this is the same across both Stripe and WCPay, so what we decide to do here, I will also do in Stripe. For now I just added the `box-shadow` back in Stripe.

Reply:
Good point about the accessibility issue.

I removed the `box-shadow` because the Google Pay button that Stripe renders (when you have the wallet available) doesn't have a `box-shadow`.

Apparently, it uses the same styling as `:hover`, so perhaps the right thing to do is to use:

```scss
&:hover, &:focus {
	background-color: #f8f8f8;
}
```

![image](https://user-images.githubusercontent.com/5509901/116766800-c19ed300-aa02-11eb-98ca-01099da38d31.png)

Reply:
We can remove the `box-shadow` styling from `:focus` in the `light-outline` theme too. Stripe adds the same styling as in `:hover`.

![image](https://user-images.githubusercontent.com/5509901/116767011-e8a9d480-aa03-11eb-8bbe-cad9414bdced.png)

Reply:
The Stripe plugin version of this is not the best version of this either IMO, so I'd really like to get @LevinMedia's 👀 on this 🙂 

I'll ping him in Slack 👍